src/qtiItem/helper/simpleParser.js

```diff
@@ -102,41 +122,68 @@ function buildTooltip(targetHtml, contentId, contentHtml) {
     };
 }
 
+function buildTable($elt, elt, options) {
+    elt.body = {
+        body: '',
+        elements: {}
+    };
+
+    const $parsedTable = parseContainer($elt, options);
+    elt.body.body = $parsedTable.body;
+    elt.body.elements = $parsedTable.elements;
+    return elt;
+}
+
 function parseContainer($container, options) {
-    var ret = {
+    const ret = {
         serial: util.buildSerial('_container_'),
         body: '',
         elements: {}
     };
 
-    _.each(_parsableElements, function(qtiClass) {
-        $container.find(qtiClass).each(function() {
-            var $qtiElement = $(this);
-            var element = buildElement($qtiElement, options);
+    $container.find('table').each(function () {
+        const $qtiElement = $(this);
+        let element = buildElement($qtiElement, options);
 
-            ret.elements[element.serial] = element;
-            $qtiElement.replaceWith(_placeholder(element));
-        });
+        element = buildTable($qtiElement, element, options);
+        ret.elements[element.serial] = element;
+        $qtiElement.replaceWith(_placeholder(element));
+    });
+
+    $container.find(_getElementSelector('img', options.ns.image)).each(function () {
```

Comment:
Instead handling `img` and `object` separate, just use array `_parsableElements` it already contains `img` and `object`
`_.each(_parsableElements, function (qtiClass) {`

Reply:
Code optimised